rov/thrusters/Thrusters_Mock.py

```diff
@@ -0,0 +1,26 @@
+class Thrusters:
```

Comment:
Added a quick and dirty mock thrusters. This should be expanded maybe so that testing with thrustmapper can output data to act like it's working to the frontend